style.css

```diff
@@ -0,0 +1,44 @@
+body {
+font-family: 'Shadows Into Light', cursive;
+background-color: hsl(208,30%,90%);
+}
+
+h1 {
+  font-size: 52px;
+  text-transform: capitalize;
+  font-weight: bold;
+  text-align: center;
+  letter-spacing: 5px;
+  word-spacing: 20px;
+  color: hsl(234,35%,24%); 
+}
+
+h2{
+  font-size: 20px;
+  font-weight: bold;
+  text-align: center;
+  letter-spacing: 2px;
+  word-spacing: 1px;
+  color: hsl(234,35%,22%); 
+}
+
+p{
+  font-family: 'Architects Daughter', cursive;
+  font-size: 20px;
+  font-weight: normal;
+  letter-spacing: 1px;
+  text-align: center;
+  color:hsl(226,18%,40%);
+  line-height: 2;
+}
+ 
+
+
+h3{
+  font-family: 'Architects Daughter', cursive;
+  font-size: 18px;
+  font-weight: bold;
+  text-align: center;
+  letter-spacing: 0.5px;
+  color: hsl(226,18%,40%); 
+}
```

Comment:
There are no errors in your CSS 🎉, but you can simplify it by combining some of your styles. [This short video](https://www.loom.com/share/e31731a4c02047578d0d04e0439186b0) shows me remixing your project and simplifying those styles.

And this is the Glitch remix I did of your project in that video: https://glitch.com/edit/#!/remix-linmin-p1